geometry_msgs/Pose targetpose
---
sensor_msgs/CompressedImage result